src/Util/TableUtil.tsx

```diff
@@ -33,7 +33,7 @@ export default class TableUtil {
           <Input
             ref={node => { searchInput = node; }}
             placeholder={`${i18n.t('GeneralEntityTable.popupSearch')} ${dataIndex}`}
-            value={`${selectedKeys[0]}`}
+            value={selectedKeys[0]}
```

Comment:
Hm, this says:

Type 'Key' is not assignable to type 'string | number | readonly string[] | undefined'.

I like the previous thing more.

Reply:
is `selectedKeys[0] ?? ''` an option?